### ISSUE TITLE
Tests are failing with ruby-mail 2.7.1

### ISSUE THREAD

**Issue by vincentbernat** [taskjuggler/TaskJuggler#229]
Hey!

With ruby-mail 2.7.1, I get the following failure:

```
rake spec
/usr/bin/ruby2.5 /usr/bin/rspec --pattern spec/\*_spec.rb
Run options: exclude {:ruby=>#<Proc:./spec/support/spec_helper.rb:15>}
......................F.............................F...............

Failures:

  1) TaskJuggler::StatusSheetTest TaskJuggler::StatusSheetSender should have matching status sheets in body and attachment
     Failure/Error: bodySheet.should == attachedSheet

       expected: "statussheet boss 2011-03-16-00:00-+0000 - 2011-03-23-00:00-+0000 {\r\n\r\n  # Task: T1\r\n  task t1 ...   #   # Work: 100%    Remaining: 5.0d (10.0d) \r\n    #   author r2\r\n    # }\r\n  }\r\n\r\n}\r\n"
            got: "statussheet boss 2011-03-16-00:00-+0000 - 2011-03-23-00:00-+0000 {\n\n  # Task: T1\n  task t1 {\n   ...:00-+0000\n    #   # Work: 100%    Remaining: 5.0d (10.0d) \n    #   author r2\n    # }\n  }\n\n}\n" (using ==)
     # ./spec/StatusSheets_spec.rb:234:in `block (4 levels) in <class:TaskJuggler>'
     # ./spec/StatusSheets_spec.rb:231:in `each'
     # ./spec/StatusSheets_spec.rb:231:in `block (3 levels) in <class:TaskJuggler>'

  2) TaskJuggler::TimeSheets TaskJuggler::TimeSheetSender should have matching timesheets in body and attachment
     Failure/Error: bodySheet.should == attachedSheet

       expected: "timesheet r1 2011-03-14-00:00-+0000 - 2011-03-21-00:00-+0000 {\r\n\r\n  # Vacation time: 0.0%\r\n\r\...details\r\n  #   ->8-\r\n  # }\r\n\r\n  # You have no future assignments for this project!\r\n}\r\n"
            got: "timesheet r1 2011-03-14-00:00-+0000 - 2011-03-21-00:00-+0000 {\n\n  # Vacation time: 0.0%\n\n  # Tas...  Some more details\n  #   ->8-\n  # }\n\n  # You have no future assignments for this project!\n}\n" (using ==)
     # ./spec/TimeSheets_spec.rb:220:in `block (4 levels) in <class:TaskJuggler>'
     # ./spec/TimeSheets_spec.rb:217:in `each'
     # ./spec/TimeSheets_spec.rb:217:in `block (3 levels) in <class:TaskJuggler>'

Finished in 22.13 seconds (files took 0.24934 seconds to load)
68 examples, 2 failures
```

**Comment by vincentbernat** [comment]
I am using this patch:

```diff
From: Vincent Bernat <vincent@bernat.ch>
Date: Sun, 27 Jan 2019 16:33:08 +0100
Subject: tests: normalize CRLF when checking MIME attachments

---
 spec/StatusSheets_spec.rb | 4 ++--
 spec/TimeSheets_spec.rb   | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/spec/StatusSheets_spec.rb b/spec/StatusSheets_spec.rb
index 1da147e..9d7be05 100644
--- a/spec/StatusSheets_spec.rb
+++ b/spec/StatusSheets_spec.rb
@@ -229,8 +229,8 @@ EOT

       it 'should have matching status sheets in body and attachment' do
         @sss_mails.each do |mail|
-          bodySheet = extractStatusSheet(mail.parts[0].decoded)
-          attachedSheet = extractStatusSheet(mail.part[1].decoded)
+          bodySheet = extractStatusSheet(mail.parts[0].decoded).gsub /\r\n/, "\n"
+          attachedSheet = extractStatusSheet(mail.part[1].decoded).gsub /\r\n/, "\n"
           bodySheet.should == attachedSheet
         end
       end
diff --git a/spec/TimeSheets_spec.rb b/spec/TimeSheets_spec.rb
index 9360130..f649ae0 100644
--- a/spec/TimeSheets_spec.rb
+++ b/spec/TimeSheets_spec.rb
@@ -215,8 +215,8 @@ EOT

       it 'should have matching timesheets in body and attachment' do
         @tss_mails.each do |mail|
-          bodySheet = extractTimeSheet(mail.parts[0].decoded)
-          attachedSheet = extractTimeSheet(mail.part[1].decoded)
+          bodySheet = extractTimeSheet(mail.parts[0].decoded).gsub /\r\n/, "\n"
+          attachedSheet = extractTimeSheet(mail.part[1].decoded).gsub /\r\n/, "\n"
           bodySheet.should == attachedSheet
         end
       end
```

If it's deemed good enough, I'll do a PR.

**Comment by scrapper** [comment]
This has been fixed for 2.7.1. Thanks for reporting it and providing a patch!
